cognite/neat/explorer/explorer.py

```diff
@@ -54,12 +51,14 @@
 configure_logging(config.log_level, config.log_format)
 logging.info(f" Starting NEAT version {neat.__version__}")
 logging.debug(f" Config: {config.dict(exclude={'cdf_client': {'client_secret': ...}})}")
-cdf_client = get_cognite_client_from_config(config.cdf_client)
-
 
 prom_app = make_asgi_app()
 
 app = FastAPI(title="Neat")
+
+neat_app = NeatApp(config, app)
+neat_app.start()
```

Comment:
This still instantiates the CDF client when deploying the app. I think we should use `Lifespan` Events instead: https://fastapi.tiangolo.com/advanced/events/#lifespan

Reply:
Will change to Lifespan and add CdfClient as optional parameter for NeatApp that will be used if set , otherwise NeatApp will instantiate new based on configs.